.travis.yml

```diff
@@ -3,7 +3,10 @@ cache: bundler
 language: ruby
 dist: trusty
 rvm:
-  - jruby-9.0.1.0
+  - jruby-9.0.5.0
+  - jruby-9.1.9.0
```

Comment:
JRuby 9.1.12 is actually available on Travis CI. For example rails/rails uses it.

https://travis-ci.org/rails/rails/jobs/271990276#L585-L586

Reply:
Thanks, updated. 

I went based on what is listed on http://rubies.travis-ci.org/. This list doesn't appear to be fully up to date.

Reply:
That's right. I don't know when list of `http://rubies.travis-ci.org/` will be updated, but as far as I know Travis CI can use the new JRuby version based on RVM update.